from https://gitlab.alpinelinux.org/alpine/aports/-/commit/eac1c6220f1e95f23af11941ce3362494a5ed3a7

The check-genfile target only works when run from inside a Git
repository. Since we build ctags from a tagged tarball and not from the
Git repository, invoking `git diff` fails with a usage error and causes
`make check` to fail.

This should be reported upstream.

diff -upr a/makefiles/testing.mak b/makefiles/testing.mak
--- a/makefiles/testing.mak	2021-10-02 15:48:41.000000000 +0200
+++ b/makefiles/testing.mak	2021-10-10 12:07:32.081801888 +0200
@@ -5,7 +5,7 @@ EXTRA_DIST += misc/units misc/units.py m
 EXTRA_DIST += misc/tlib misc/mini-geany.expected
 MAN_TEST_TMPDIR = ManTest
 
-check: tmain units tlib man-test check-genfile
+check: tmain units tlib man-test
 
 # We may use CLEANFILES, DISTCLEANFILES, or etc.
 # clean-tlib and clean-gcov are not included
